Lib/platform.py

```diff
@@ -1156,9 +1156,10 @@ def _sys_version(sys_version=None):
     sys_version_parser = re.compile(
         r'([\w.+]+)\s*'  # "version<space>"
         r'\(#?([^,]+)'  # "(#buildno"
-        r'(?:,\s*([\w ]*)'  # ", builddate"
-        r'(?:,\s*([\w :]*))?)?\)\s*'  # ", buildtime)<space>"
-        r'\[([^\]]+)\]?', re.ASCII)  # "[compiler]"
+        r'(?:,\s*([\w ]*)'  # ", buildate"
+        r'(?:,\s*([\w :]*))?)?'  # ", buildtime"
+        r'(?:,\s*(free-threading))?\)*\s*'  # ",'free-threading')<space>'"
+        r'\[([^\]]+)\]?', re.ASCII)  #"[compiler]"
```

Comment:
Fix typo, and restore removed space:

```suggestion
        r'(?:,\s*([\w ]*)'  # ", builddate"
        r'(?:,\s*([\w :]*))?)?'  # ", buildtime"
        r'(?:,\s*(free-threading))?\)*\s*'  # ",'free-threading')<space>'"
        r'\[([^\]]+)\]?', re.ASCII)  # "[compiler]"
```